File: client/lib/poll.dart
class Poll {
  final String question;
  final List<String> options;
  final List<int> votes;

  Poll({required this.question, required this.options})
      : votes = List<int>.filled(options.length, 0);

  void vote(int index) {
    votes[index]++;
  }
}

File: backend/models/poll.js
const mongoose = require('mongoose');
const Schema = mongoose.Schema;

const pollSchema = new Schema({
  question: { type: String, required: true },
  description: { type : String},
  options: [
    {
      option: { type: String, required: true }
    }
  ],
  createdAt: { type: Date, default: Date.now },
  endDate: { type: Date, default: null },
  status: {type: Boolean, default: true}
});
//hhnhnjnjnj
const Poll = mongoose.model('Poll', pollSchema);
module.exports = Poll;
